source/GameData.cpp

```diff
@@ -187,7 +187,16 @@ void GameData::LoadShaders(bool useShaderSwizzle)
 
 double GameData::GetProgress()
 {
-	return min(min(spriteQueue.GetProgress(), Audio::GetProgress()), objects.GetProgress());
+	// Cache progress completion seen, so clients are
+	// isolated from the loading implementation details.
+	static bool initiallyLoaded;
```

Comment:
```suggestion
	static bool initiallyLoaded = false;
```

I know it's the default, but we want to be clear.